BFS_Connected_Components/Q10_Leetcode_number_of_closed_islands.py

```diff
@@ -0,0 +1,38 @@
+import collections
+def closedIsland(grid):
+    ROWS, COLS = len(grid), len(grid[0])
+    
+    directions = [[0, 1], [0, -1], [1, 0], [-1, 0]]
+    seen = set()
+    
+
+    def is_closed_island(r, c):
+        q = collections.deque([(r, c)])
```

Comment:
You should avoid one letter variable names in pretty much all the situations (except i and j in loops). Name this `queue`.